src/Compilers/CSharp/Portable/Binder/ExpressionVariableFinder.cs

```diff
@@ -13,8 +13,9 @@ namespace Microsoft.CodeAnalysis.CSharp
     internal class ExpressionVariableFinder : CSharpSyntaxWalker
     {
         private Binder _scopeBinder;
-        private Binder _enclosingBinderOpt;
+        private Binder _enclosingBinder;
         private ArrayBuilder<LocalSymbol> _localsBuilder;
+        private SyntaxNode _nodeToBind;
```

Comment:
>_nodeToBind [](start = 27, length = 11)

Is deconstruction going to be properly handled by this visitor with respect to _nodeToBind?